js/packages/openinference-instrumentation-openai/test/openai.test.ts

```diff
@@ -421,34 +422,273 @@ describe("OpenAIInstrumentation", () => {
+              created: 1705535755,
+              model: "gpt-3.5-turbo-0613",
+              system_fingerprint: null,
+              choices: [
+                {
+                  index: 0,
+                  delta: {},
+                  logprobs: null,
+                  finish_reason: "function_call",
+                },
+              ],
+            };
+          })();
+        const controller = new AbortController();
+        return new Stream(iterator, controller);
+      },
+    );
+    const stream = await openai.chat.completions.create({
+      messages: [{ role: "user", content: "What's the weather today?" }],
+      model: "gpt-3.5-turbo",
+      functions: [
+        {
+          name: "getWeather",
+          description: "Get the weather for a location.",
+          parameters: {
+            type: "object",
+            properties: {
+              location: { type: "string" },
+            },
+          },
+        },
+        {
+          name: "getCurrentLocation",
+          description: "Get the current location of the user.",
+          parameters: { type: "object", properties: {} },
+        },
+      ],
+      stream: true,
+    });
+
+    let response = "";
+    for await (const chunk of stream) {
+      if (chunk.choices[0].delta.content)
+        response += chunk.choices[0].delta.content;
+    }
+    // When a tool is called, the content is empty
+    expect(response).toBe("");
+    const spans = memoryExporter.getFinishedSpans();
+    expect(spans.length).toBe(1);
+    const span = spans[0];
+    expect(span.name).toBe("OpenAI Chat Completions");
+    expect(span.attributes).toMatchInlineSnapshot(`
+      {
+        "input.mime_type": "application/json",
+        "input.value": "{"messages":[{"role":"user","content":"What's the weather today?"}],"model":"gpt-3.5-turbo","functions":[{"name":"getWeather","description":"Get the weather for a location.","parameters":{"type":"object","properties":{"location":{"type":"string"}}}},{"name":"getCurrentLocation","description":"Get the current location of the user.","parameters":{"type":"object","properties":{}}}],"stream":true}",
+        "llm.input_messages.0.message.content": "What's the weather today?",
+        "llm.input_messages.0.message.role": "user",
+        "llm.invocation_parameters": "{"model":"gpt-3.5-turbo","functions":[{"name":"getWeather","description":"Get the weather for a location.","parameters":{"type":"object","properties":{"location":{"type":"string"}}}},{"name":"getCurrentLocation","description":"Get the current location of the user.","parameters":{"type":"object","properties":{}}}],"stream":true}",
+        "llm.model_name": "gpt-3.5-turbo",
+        "llm.output_messages.0.message.content": "",
+        "llm.output_messages.0.message.function_call_arguments_json": "{}",
+        "llm.output_messages.0.message.function_call_name": "getWeather",
+        "llm.output_messages.0.message.role": "assistant",
```

Comment:
Same as above.

Reply:
Yeah not the tool call, the first message is the assistant saying to make a tool / function call